src/cryptography/hazmat/primitives/serialization/ssh.py

```diff
@@ -1052,6 +1052,9 @@ def serialize_ssh_public_key(public_key: _SSH_PUBLIC_KEY_TYPES) -> bytes:
 ]
 
 
+_SSHKEY_CERT_MAX_PRINCIPALS = 256
```

Comment:
Can you please add a comment indicating that while this isn't part of the spec, it is enforced by openssh?

Reply:
Added in an explainer comment.